packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -14,6 +14,7 @@ import { Router as CatalogRouter } from '@backstage/plugin-catalog';
 import { Router as DocsRouter } from '@backstage/plugin-techdocs';
 import { Router as RegisterComponentRouter } from '@backstage/plugin-register-component';
 import { Router as TechRadarRouter } from '@backstage/plugin-tech-radar';
+import { SearchPage as SearchRouter } from '@backstage/plugin-search'
```

Comment:
```suggestion
import { SearchPage as SearchRouter } from '@backstage/plugin-search';
```

Reply:
Oh - good catch @OrkoHunter ! Seems like I have an issue with Prettier here - I will fix that for the future. Thanks!